src/api/external/cristinProjectApi.ts

```diff
@@ -0,0 +1,27 @@
+import Axios from 'axios';
+import { ExternalApi, StatusCode } from '../../utils/constants';
+import { addNotification } from '../../redux/actions/notificationActions';
+import { Dispatch } from 'redux';
+import i18n from '../../translations/i18n';
+import uuid from 'uuid';
+import { Notification } from '../../types/notification.types';
+
+const errorNotification: Notification = {
+  key: uuid.v4(),
+  message: i18n.t('feedback:error.get_cristin_project'),
+  variant: 'error',
+};
+
+export const searchCristinProjects = async (query: string, dispatch: Dispatch) => {
+  try {
+    const response = await Axios.get(`${ExternalApi.CRISTIN}/projects?${query}`);
+    if (response.status === StatusCode.OK && response.headers) {
+      response.data.totalCount = response.headers['X-Total-Count'];
+      return response.data;
+    } else {
+      dispatch(addNotification(errorNotification));
```

Comment:
var dette noe som skulle ventes på? kan evt legge til at key er optional på denne PRen også isåfall?

Reply:
Funker ikke med optional key. Kjetil ser på saken i sin PR. Kan avvente

Reply:
Endte opp med å sende inn `key: uuid.v4()` her. Men den blir sikkert uansett overskrevet av reducer, så det er en dårlig løsning. Prøvde å finne en bedre løsning, men det ble så mye ball at jeg droppet det. Tenker det er best å ha en egen PR som fikser dette